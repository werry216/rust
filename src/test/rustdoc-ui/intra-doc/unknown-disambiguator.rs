//! Linking to [foo@banana] and [`bar@banana!()`].
//~^ ERROR unknown disambiguator `foo`
//~| ERROR unknown disambiguator `bar`
//! And to [no disambiguator](@nectarine) and [another](@apricot!()).
//~^ ERROR unknown disambiguator ``
//~| ERROR unknown disambiguator ``

#![deny(warnings)]

fn main() {}
