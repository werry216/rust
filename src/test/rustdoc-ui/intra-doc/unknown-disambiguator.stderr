error: unknown disambiguator `foo`
  --> $DIR/unknown-disambiguator.rs:1:17
   |
LL | //! Linking to [foo@banana] and [`bar@banana!()`].
   |                 ^^^^^^^^^^
   |
note: the lint level is defined here
  --> $DIR/unknown-disambiguator.rs:8:9
   |
LL | #![deny(warnings)]
   |         ^^^^^^^^
   = note: `#[deny(rustdoc::broken_intra_doc_links)]` implied by `#[deny(warnings)]`
   = note: the disambiguator is the part of the link before the `@` sign, or a suffix such as `()` for functions

error: unknown disambiguator `bar`
  --> $DIR/unknown-disambiguator.rs:1:34
   |
LL | //! Linking to [foo@banana] and [`bar@banana!()`].
   |                                  ^^^^^^^^^^^^^^^
   |
   = note: the disambiguator is the part of the link before the `@` sign, or a suffix such as `()` for functions

error: unknown disambiguator ``
  --> $DIR/unknown-disambiguator.rs:4:31
   |
LL | //! And to [no disambiguator](@nectarine) and [another](@apricot!()).
   |                               ^^^^^^^^^^
   |
   = note: the disambiguator is the part of the link before the `@` sign, or a suffix such as `()` for functions

error: unknown disambiguator ``
  --> $DIR/unknown-disambiguator.rs:4:57
   |
LL | //! And to [no disambiguator](@nectarine) and [another](@apricot!()).
   |                                                         ^^^^^^^^^^^
   |
   = note: the disambiguator is the part of the link before the `@` sign, or a suffix such as `()` for functions

error: aborting due to 4 previous errors

